docs/tutorial/v1/guestbook-statefulset-v2.yaml

```diff
@@ -1,37 +0,0 @@
-apiVersion: apps.kruise.io/v1alpha1
-kind: StatefulSet
-metadata:
-  name: guestbook-v1
-  labels:
-    app: guestbook
-    version: "1.0"
-spec:
-  replicas: 20
-  serviceName: guestbook
-  selector:
-    matchLabels:
-      app: guestbook
-  template:
-    metadata:
-      labels:
-        app: guestbook
-        version: "1.0"
-    spec:
-      readinessGates:
-        # A new condition that ensures the pod remains at NotReady state while the in-place update is happening
-        - conditionType: InPlaceUpdateReady
-      containers:
-      - name: guestbook
-        image: openkruise/guestbook:v2
-        ports:
-        - name: http-server
-          containerPort: 3000
-  podManagementPolicy: Parallel  # allow parallel updates, works together with maxUnavailable
-  updateStrategy:
-    type: RollingUpdate
-    rollingUpdate:
-      # Do in-place update if possible, currently only image update is supported for in-place update
```

Comment:
why remove this file, it's used in the tutorial for in-place update

Reply:
The file has been renamed to `https://github.com/openkruise/kruise/blob/master/docs/tutorial/v1/guestbook-patch-to-v2.yaml`,  `guestbook-statefulset-v2.yaml` is not used for long time.